test/e2e/framework/output_handling.go

```diff
@@ -123,3 +123,9 @@ type PluginDescribe struct {
 	Target                       string `yaml:"target"`
 	Version                      string `yaml:"version"`
 }
+
+type InputConfigData struct {
+	PluginsForLifeCycleTests      []*PluginInfo  `json:"PluginsForLifeCycleTests"`
+	PluginGroupsForLifeCycleTests []*PluginGroup `json:"PluginGroupsForLifeCycleTests"`
+	EssentialPluginGroups         []*PluginGroup `json:"EssentialPluginGroups"`
+}
```

Comment:
Same as above: This does not seem like something that belongs under `Framework`. This is specific to the Tanzu CLI Core tests and should not be part of the Framework code.